state/indexer/query_range.go

```diff
@@ -67,7 +69,9 @@ func (qr QueryRange) UpperBoundValue() interface{} {
 	switch t := qr.UpperBound.(type) {
 	case int64:
 		return t - 1
-
+	case *big.Int:
```

Comment:
(from sync discussion) Do we need to delete 70-71 to be consistent with `LowerBoundValue`

Reply:
Reverted the deletion in `LowerBoundValue`